airbyte-webapp/src/views/Connector/ConnectorForm/useBuildForm.tsx

```diff
@@ -54,19 +70,18 @@ interface BuildUiWidgetsContextHook {
 
 export const useBuildUiWidgetsContext = (
   formFields: FormBlock[] | FormBlock,
-  formValues: ConnectorFormValues,
-  uiOverrides?: WidgetConfigMap
+  formValues: ConnectorFormValues
 ): BuildUiWidgetsContextHook => {
-  const [overriddenWidgetState, setUiWidgetsInfo] = useState<WidgetConfigMap>(uiOverrides ?? {});
+  const [overriddenWidgetState, setUiWidgetsInfo] = useState<WidgetConfigMap>({});
```

Comment:
Since there is no more concept of "ui overrides" here, I think we can just name this something like `uiWidgetsInfo` instead of `overriddenWidgetState`

Reply:
I will take care of this in a follow-up PR